src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,52 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private final StorageItem[] storageItems;
+    private int indexCounter = 0;
```

Comment:
* this is kind of obvious that it is a number :)
* primitive class fields have default values
```suggestion
    private static final int MAX_ITEMS = 10;
    private final Pair[] pairs;
    private int size;
```